src/_includes/community/requesting-new-icons.html

```diff
@@ -23,7 +23,7 @@ <h2 class="page-header">Requesting New Icons</h2>
       </ul>
     </li>
     <li>
-      Request concrete objects: it's harder to make an icon to represent happiness, it's easier to make a smiley face.  <i class="fa fa-smile-o"></i>
+      Request concrete objects: it's harder to make an icon to represent happiness, it's easier to make a smiley face.  <i class="fa fa-smile-o" aria-hidden="true"></i>
```

Comment:
I would argue that this example is not purely decorative and therefore needs an alt text